admin-manual/installation/installation.rst

```diff
@@ -608,7 +474,250 @@ If IPv6 is disabled, Nginx may refuse to start. If that is the case make sure th
+you can create a virtual machine and follow the manual install instructions.  
+
+The ansible roles referenced here can be used in production deployments 
+by creating your own ansible playbook to run them. See 
+https://github.com/artefactual/deploy-pub/playbooks/archivematica for more 
+details.
+
+1. Install Dependencies
+
+These instructions require VirtualBox, Vagrant, and Ansible
+
+   .. code:: bash
+
+      sudo apt-get install virtualbox vagrant
+      sudo pip install -U ansible
+
+   Vagrant must be at least version 1.5. Check your version with:
+
+   .. code:: bash
+
+      vagrant --version
+
+   If it is not up to date, you can download the newest version from the 
+   `Vagrant website <https://www.vagrantup.com/downloads.html>`_ .
+
+2. Download Installer
+
+Checkout the deployment repo:
+
+   .. code:: bash
+
+      git clone https://github.com/artefactual/deploy-pub.git
+
+3. Dependencies
+
+Download the Ansible roles:
+
+   .. code:: bash
+
+      cd deploy-pub/playbooks/archivematica
+      ansible-galaxy install -f -p roles/ -r requirements.yml
+
+4. Install
+
+Create the virtual machine and provision it:
+
+   .. code:: bash
+
+      vagrant up
+
+   .. warning::
+
+     This will take a while.
+     It depends on your computer, but it could take up to an hour.
+     Your computer may be very slow while Archivematica is being provisioned - 
+     be sure to save any work and be prepared to step away from your computer 
+     while Archivematica is building.
+
+5. Re-provisioning
+
+If there's an error, you can re-run the setup.
+
+   .. code:: bash
+
+      vagrant provision
+
+Once it's done provisioning, you can log in to your virtual machine::
+
+  vagrant ssh
+
+You can also access your Archivematica instance through the web browser:
+
+* Archivematica: `<http://192.168.168.192>`_. Username & password configured on 
+  installation.
+* Storage Service: `<http://192.168.168.192:8000>`_. Username: test, password: 
+  test.
+
+6. Post Install Configuration
+
+See :ref:`Post Install Configuration <post-install-config>`
+
+
+.. _post-install-config:
+
+Post Install Configuration
+--------------------------
+
+After successfully completing a new installation using one of the methods 
+above, follow these steps to complete the configuration of your new server. 
+
+1. Test the storage service
+
+The storage service runs as a separate web application from the Archivematica 
+dashboard. Go to the following link in a web browser and log in as user *test* 
+with the password *test*: http://localhost:8000 (or use the IP address of the
+machine you have been installing on).
+
+2. New Storage Service User
+
+Create a new administrative user in the Storage service. The storage service 
+has its own set of users. In the User menu in the Administrative tab of the 
+storage service, add at least one administrative user, and delete or modify the
+test user. After you have created an administrative user, copy its API key to 
```

Comment:
I think it actually has to be modify- not currently possible to delete users in the SS.